core/src/dird/ndmp_dma_storage.cc

```diff
@@ -634,7 +666,10 @@ bool NdmpUpdateStorageMappings(UaContext* ua, StorageResource* store)
 {
   struct ndm_session* ndmp_sess;
 
-  if (!GetRobotElementStatus(ua->jcr, store, &ndmp_sess)) { return false; }
+  if (!GetRobotElementStatus(ua->jcr, store, &ndmp_sess)) {
+    Dmsg0(200, "Could not get GetRobotElementStatus.\n");
```

Comment:
I think this should be identical to the wording in line 651:
Dmsg0(200, "Could not get robot element status.\n");